client/src/components/dropdown-group.js

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 import { ButtonToolbar } from 'react-bootstrap';
 import Button from 'react-bootstrap/Button';
+import PropTypes from 'prop-types';
 import DropdownWithDate from './custom-dropdown';
 import MyDropdown from './my-dropdown';
+import CategoryDropdown from './category-dropdown';
 
 class DropdownGroup extends React.Component {
+  onSelectCategory(path) {
+    console.log(`path= ${path}`);
```

Comment:
Is this `console.log` need?

Reply:
You right it's not needed

Reply:
Ok! Than I'll approve PR after you'll remove it. Your code works very well!